src/utils/convertStrappedToUpswyng.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  TResource,
+  TUpswyngResource,
+  TFirebaseTimestamp,
+  TSchedule
+} from '../types';
+
+function convertStrappedDateToFirebaseTimestamp(
+  date: string
+): TFirebaseTimestamp {
+  const jsDate = new Date(date);
+  const secondsStamp: number =
+    jsDate instanceof Date &&
+    !isNaN(jsDate.getTime()) /* tests for valid date */
+      ? Math.round(
+          (jsDate.getTime() + jsDate.getTimezoneOffset() * 60000) / 1000
+        )
+      : 0;
+  return { _seconds: secondsStamp, _nanoseconds: 0 };
+}
+
+function convertResource(charity: TResource): TUpswyngResource {
+  return {
+    address: {
+      address1: charity.address1,
+      address2: charity.address2,
+      city: charity.city,
+      state: charity.state,
+      zip: charity.zip
+    },
+    category: charity.category,
```

Comment:
May be outside the scope of this PR, but this currently contains a string of groups the resources serves, e.g., `"Kids,LGBT+,Families"` etc. It may be useful to spread these into something like:
```
groups: [
  'Kids',
  'LGBT+',
  'Families',
]
```